.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "bin/yapf"]
 	path = bin/yapf
 	url = https://github.com/google/yapf.git
+[submodule "src/libs/base64"]
```

Comment:
Yep, removed the submodule and just copied the header. Ported the tests to `base64_tests.cpp`.